src/nvim/fileio.c

```diff
@@ -5992,18 +6011,20 @@ void ex_doautoall(exarg_T *eap)
   check_cursor();           /* just in case lines got deleted */
 }
 
-/*
- * Check *argp for <nomodeline>.  When it is present return FALSE, otherwise
- * return TRUE and advance *argp to after it.
- * Thus return TRUE when do_modelines() should be called.
- */
-int check_nomodeline(char_u **argp)
+/// Check *argp for <nomodeline>.  When it is present return false, otherwise
+/// return true and advance *argp to after it.
+///
+/// @param[in,out] argp argument string
+///
+/// @return true when do_modelines() should be called.
```

Comment:
I think this split-off was a bit unfortunate. Now both the description and `@return` talk about retval, but say different things. I would suggest just reverting this split, maybe changing order to `Thus do_modelines() should be called when true is returned` also.